client/components/settings/local_settings_modal.html

```diff
@@ -0,0 +1,21 @@
+<div class="ui right floated basic icon compact item" ng-click="localSettingsModal.modalVisible = true; localSettingsModal.loadSettings()">
+		<i class="cog icon"></i>
+		<div class="hover text">Settings</div>
+</div>
+<sm-modal class="longer small settingsmodal" visible="localSettingsModal.modalVisible" settings="{ closable: true }">
```

Comment:
Seperate the words int `settingsmodal`, it's not one word. It should read `settings-modal`.